### PR TITLE
Update ag_logging.py with clarity and reference

### PR DESCRIPTION
**Pull request by ghchinoy**
* fixes typo
* adds clarity to text
* adds reference to absl's logging doc page

ref #29250 #29251

### CODE REVIEW

File: tensorflow/python/autograph/utils/ag_logging.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def set_verbosity(level, alsologtostdout=False):
   More verbose logging is useful to enable when filing bug reports or doing
   more in-depth debugging.
 
-  There are two controls that control the logging verbosity:
+  There are two parameters that control the logging verbosity:
```
Comment (mdanatg) [inline tensorflow/python/autograph/utils/ag_logging.py]: How about; "two means to control"

**Review by mdanatg — DISMISSED** [review]
Thanks for the fixes!

File: tensorflow/python/autograph/utils/ag_logging.py (hunk 1 above)
Comment (ghchinoy) [reply tensorflow/python/autograph/utils/ag_logging.py]: lgtm

Comment (mdanatg) [reply tensorflow/python/autograph/utils/ag_logging.py]: Great, please update the PR then. Thanks!

**Comment by gbaned** [comment]
@ghchinoy  Could you please address the reviewer comments. Thanks!

**Review by mdanatg — APPROVED** [review]

**Pull request merged**